rules/windows/credential_access_spn_attribute_modified.toml

```diff
@@ -0,0 +1,126 @@
+
+- Identify the account that performed the action.
+- Check whether this user should be doing this kind of activity.
+- Contact the account owner and confirm whether they are aware of this activity.
+- Investigate if the target account is a member of Privileged groups (Domain Admins, Enterprise Admins, etc).
+- Investigate if tickets have been requested for the target account.
+- Investigate other alerts related to the user in the last 48 hours.
+
+### False Positive Analysis
+
+- The use of user accounts as service accounts is a bad security practice and should not be allowed in the domain. The
+security team should map and monitor any potential B-TP (Benign True Positive), especially if the account is privileged.
+Domain Administrators that define this kind of setting can put the domain at risk as user accounts don't have the same
+security standards (Long, complex, random passwords that change frequently) as computer accounts, exposing them to
+credential cracking attacks (Kerberoasting, brute force, etc.).
+
+### Response and Remediation
+
+- Initiate the incident response process based on the outcome of the triage
+- Reset the password of the involved accounts. Priority should be given to privileged accounts.
+- Quarantine the involved host for forensic investigation, as well as eradication and recovery activities.
+
+## Config
+
+The 'Audit Directory Service Changes' logging policy must be configured for (Success, Failure).
+Steps to implement the logging policy with Advanced Audit Configuration:
+
+```
+Computer Configuration > 
+Policies > 
+Windows Settings > 
+Security Settings > 
+Advanced Audit Policies Configuration > 
+Audit Policies > 
+DS Access > 
+Audit Directory Service Changes (Success,Failure)
+```
+
+The above policy does not cover User objects, so we need to set up an AuditRule using https://github.com/OTRF/Set-AuditRule.
+As this specifies the servicePrincipalName Attribute GUID, it is expected to be low noise.
+
+```
+Set-AuditRule -AdObjectPath 'AD:\\CN=Users,DC=Domain,DC=com' -WellKnownSidType WorldSid -Rights WriteProperty -InheritanceFlags Children -AttributeGUID f3a64788-5306-11d1-a9c5-0000f80367c1 -AuditFlags Success
+```
+"""
+references = [
+    "https://www.thehacker.recipes/ad/movement/access-controls/targeted-kerberoasting",
+    "https://www.qomplx.com/qomplx-knowledge-kerberoasting-attacks-explained/",
+    "https://www.thehacker.recipes/ad/movement/kerberos/kerberoast",
+    "https://serverfault.com/questions/541035/kerberos-adding-a-spn-to-a-domain-user",
+    "https://attack.stealthbits.com/cracking-kerberos-tgs-tickets-using-kerberoasting",
+    "https://adsecurity.org/?p=280",
+    "https://github.com/OTRF/Set-AuditRule",
+]
+risk_score = 73
+rule_id = "0b2f3da5-b5ec-47d1-908b-6ebb74814289"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access", "Active Directory"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
```

Comment:
We may be able to add a check to see if the user is associated to a domain with `SubjectDomainName`. As highlighted in the [qomplx](https://www.qomplx.com/qomplx-knowledge-kerberoasting-attacks-explained/#how-kerberoasting-attacks-work) link you provided, this attack will really only work against domain user SPNs.

Reply:
I can't see a scenario where an SPN could be assigned to a local account, as it is a feature of Active Directory.